lib/improver/tests/wind_downscaling/test_RoughnessCorrection.py

```diff
@@ -646,6 +648,12 @@ def test_section1f(self):
         land_hc_rc = landpointtests_hc_rc.run_hc_rc(self.uin)
         self.assertArrayEqual(landpointtests_hc_rc.w_cube, land_hc_rc)
 
+    def test_section1g(self):
+        """ Test that code returns float32 precision. """
```

Comment:
Please remove the space after the first triple quotes - this is to do with how the documentation renders in Sphinx (see https://github.com/metoppv/improver/wiki/Code-Style-Guide#unit-test-structure)